handlers/amazonec2_test.go

```diff
@@ -50,7 +50,7 @@ func TestAmazonec2(t *testing.T) {
 func setup(accessKey string, secretKey string, vpcId string, zone string) {
 	// TODO Replace functions during teardown.
 	machine := &client.Machine{
-		Amazonec2Config: client.Amazonec2Config{
+		Amazonec2Config: &client.Amazonec2Config{
```

Comment:
@wlan0 Oh. hm. The build failed and it looks related to this change. Do you still need to push the godep updates?